README.md

```diff
@@ -177,6 +177,7 @@ linux/arm64
 
 * `RSPAMD_ENABLE`: Enable Rspamd service. (default `false`)
 * `RSPAMD_WEB_PASSWORD`: Rspamd web password (default `null`)
+* `RSPAMD_DISABLE_WEB_PASSWORD`: Disable Rspamd web password (default `false`)
```

Comment:
Maybe less confusing to name it `RSPAMD_SECURE_IP` that defaults to `127.0.0.1/32`

Reply:
I do authentication myself via [forward_auth](https://caddyserver.com/docs/caddyfile/directives/forward_auth) and Authelia on my reverse proxy. The configuration on master doesn't let me disable the web password so clients authenticated via my reverse proxy still have to enter the web password. This PR allows the web password to be disabled.

Allowing for direct control of the `secure_ip` parameter is a good idea. I've updated it accordingly.